frontend/src/auth/components/PasswordForgottenPage/PasswordForgottenPage.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import { RootState } from 'global/state';
+import { connect } from 'react-redux';
+import { WrappedMessage } from 'utils/intl';
+import { Alert, Button, Spinner, Form } from 'react-bootstrap';
+import { ContentPage, TextInputField } from 'ui/components';
+import { performPasswordForgotten } from 'auth/actions';
+import { Link } from 'react-router-dom';
+import messages from './displayMessages';
+import './styles.scss';
+import { ROUTES } from '../../../global/constants';
+
+interface ActionProps {
+  performPasswordForgotten: Function;
+}
+
+interface State {
+  [key: string]: string;
+  email: string;
+}
+
+interface Props extends ActionProps {
+  loading: boolean;
+  succeeded: boolean;
+  error: string;
+}
+
+@connect<{}, ActionProps, {}, Props, RootState>(
+  (state: RootState) => state.loginState,
+  {
+    performPasswordForgotten
+  }
+)
+export class PasswordForgottenPage extends React.PureComponent<Props, State> {
+  constructor(props: Props, state: State) {
+    super(props);
+
+    this.state = {
+      email: ''
+    };
+  }
+
+  private onKeyPress = (event: any) => {
+    if (event.key === 'Enter') {
+      event.preventDefault();
+      this.submit();
+    }
+  };
+
+  private submit = () => {
+    this.props.performPasswordForgotten({ email: this.state.email });
+  };
+
+  private onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const field = e.target.name;
+    const { value } = e.target;
+
+    this.setState({
+      [field]: value
+    });
+  };
+
+  public render() {
+    return (
+      <ContentPage title="Forgot password">
+        <Form
+          className="login" // FIXME: make SCSS more generic to properly reuse this style.
+          onKeyPress={(event: any) => {
+            this.onKeyPress(event);
+          }}
+        >
+          {this.props.succeeded && (
+            <Alert variant="success">
+              <WrappedMessage messages={messages} id="success" />
+            </Alert>
+          )}
+
+          <TextInputField
+            fieldName="email"
+            value={this.state.email}
+            onChange={this.onChange}
+            messages={messages}
+            type="email"
+          />
+
+          {this.props.error && (
+            <Alert variant="danger">{this.props.error}</Alert>
```

Comment:
Nit: Can you check if it's possible to add the styles from the mockups to these alerts for the entire app (with a generic selector)? 

![danger](https://user-images.githubusercontent.com/27893385/80016780-fac86d00-84a9-11ea-9213-52ee7014048f.png)
![success](https://user-images.githubusercontent.com/27893385/80016783-fc923080-84a9-11ea-91dd-12df1582e532.png)

Reply:
Done.

Reply:
@Agrendalath They need some tweaking. 
I thought you had access to the styles from Zeplin, let me know if that's not the case.

Here are the styles:
**Danger**
* Font:
```
  font-family: Chivo;
  font-size: 14px; // You can adjust this (probably 12px is going to look closer to the mockups)
  color: #ffffff;
```
* BG:
```
background-color: #dd0000;
```

**Success**
* Font:
```
  font-family: Chivo;  // $chivo-font if you import the global styles
  font-size: 16px;  // Adjust as you see fit, probably having the same for both is better
  color: #ffffff;
```
* BG:
```
 background-color: #00a556;
```

Reply:
@giovannicimolin, you didn't have enough privileges to give me this access ([cf. this comment](https://tasks.opencraft.com/browse/BB-2219?focusedCommentId=143230&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-143230)). Though agreed, it would be nice to have.

Reply:
Fixed. IMO 14px looked well-balanced for both.